components/html-block/html-block.js

```diff
@@ -231,7 +233,9 @@ class HtmlBlock extends LitElement {
 		};
 
 		return html`
-			<div class="${classMap(renderContainerClasses)}"></div>
+			<div class="${classMap(renderContainerClasses)}">
+				${!this.noDeferredRendering ? until(this._processEmbeds(), unsafeHTML(this.html)) : nothing}
```

Comment:
That's right. The alternative I guess would be to render nothing. I'm not entirely sure which is preferable. My gut feeling was that rendering the actual HTML would be less jittery than just rendering nothing, but I don't have strong feelings on that. Rendering nothing would probably look a bit more like it just took a longer time to render in the first place.

Reply:
We may have to test it in practice to find out. I'm worried about "slow" things like images, videos, etc. that start downloading, then get removed, re-added and download again.

Reply:
That's a good point. To make things worse, this is already going to happen once because MathJax needs to remove everything from the DOM and re-add it here: https://github.com/BrightspaceUI/core/blob/main/helpers/mathjax.js#L69

So minimizing how often we need to completely re-render content is definitely ideal.